vizro-core/examples/_chart-gallery/pages/examples/heatmap.py

```diff
@@ -0,0 +1,15 @@
+import vizro.models as vm
+import vizro.plotly.express as px
+from vizro import Vizro
+
+tips = px.data.tips()
+
+page = vm.Page(
+    title="Heatmap",
+    components=[
+        vm.Graph(figure=px.density_heatmap(tips, x="day", y="size", z="tip", histfunc="avg", text_auto="$.1f")),
```

Comment:
```suggestion
        vm.Graph(figure=px.density_heatmap(tips, x="day", y="size", z="tip", histfunc="avg", text_auto="$.2f")),
```
Doing money as anything other than 2 decimal places looks very weird.

Do we definitely want the `$` in every cell rather than in the colorbar title?

Reply:
I wanted to prevent that, because to customise the color bar title one has to use an update call 😬 Let me just remove the $.

Reply:
Ah ok, I thought that might have been it. I don't mind the $ in every cell - probably better than not having it at all. So I'd just leave it there as you have it and make sure there's 2 decimal places.